modules/memberships/class-jetpack-memberships.php

```diff
@@ -389,10 +459,13 @@ public function register_gutenberg_block() {
 		}
 
 		if ( self::is_enabled_jetpack_recurring_payments() ) {
+			$deprecated = function_exists( 'gutenberg_get_post_from_context' );
```

Comment:
How far back does this go? Jetpack now requires WordPress 5.5, so we may not need that check anymore. I've tried to figure that out by `blame`ing `lib/compat.php` in Gutenberg, but I may not be looking in the right place.

Reply:
I tried searching for this function in the source for WP5.5 and could not find it, so I think this might still be necessary.